typescript/hx-adl/src/gen-sqlschema.ts

```diff
@@ -41,21 +42,62 @@ export function configureCli(program: Command) {
        dbProfile = mssql2DbProfile;
      }
 
-     generateSql({adlFiles, adlSearchPath, outfile, outmetadata, extensions, templates, dbProfile});
+     generateSql({adlFiles, adlSearchPath, outfiles, outmetadata, extensions, templates, dbProfile}, generateCreateSqlSchema);
    });
 }
 
+export function configureAlterFormatSqlCli(program: Command) {
+  program
+    .command("sql-alter-format [adlFiles...]")
+    .option('-I, --searchdir <path>', 'Add to adl searchpath', collect, [])
+    // .option('--outfile <path>', 'the resulting sql file', 'create.sql')
+    .option('--outputdir <dir>', 'the directory into which the sql is written (deprecated)')
+    .option('--outmetadata <path>', 'sql to insert the model metadata')
+    .option('--outtemplatesql <paths>', 'generate extra sql from a mustache template', collect, [])
+    .option('--postgres', 'Generate sql for postgres')
+    .option('--postgres-v2', 'Generate sql for postgres (model version 2)')
+    .option('--mssql', 'Generate sql for microsoft sqlserver')
+    .option('--extension <ext>', 'Add to included sql extensions', collect, [])
+    .option('--migration', 'Generate sql for migration')
+    .description('Generate a db schema from ADL files')
+    .action((adlFiles: string[], cmd: {}) => {
+      const adlSearchPath: string[] = cmd['searchdir'];
+      const extensions: string[] = cmd['extension'];
+      const templates: Template[] = parseTemplates(cmd['outtemplatesql'] || []);
+
+      let outfileCreate: string = "Vx__create.sql"
+      let outfileConstrant: string = "Vy__constraints.sql"
```

Comment:
```suggestion
      let outfileCreate: string = "V00x_0__create.sql"
      let outfileConstrant: string = "V00x_1__constraints.sql"
```

Suggest naming so that typically a "version 009" migration would have such files named with minor version number (This is a format that flyway understands)